lca-chimevc-stack/LambdaHookFunction.md

```diff
@@ -0,0 +1,84 @@
+# Using a Lambda function to optionally provide custom logic for call handling
+
+## Overview
+
+LCA v0.5.0 and later offers optional custom logic via a user-provided Lambda function, to support the following features:
+1. Selectively choose whether to process a call, or to ignore it
+2. Reverse the default assignment of caller and agent audio streams
+3. Assign an Agent identifier string for display, search, and sort on the LCA UI.  
+   *NOTE see [SettingAgentId.md](./SettingAgentId.md) if you need to provide an AgentID after the call has started.*
+4. Override the default CallId with another unique value for the call
+5. Override the default toNumber (System Phone number)
+6. Override the default fromNumber (Caller Phone number)
+
+To use this feature:
+1. Implement a Lambda function with the desired business logic
+2. Use CloudFormation to register the Lambda with your LCA stack
+
+## Lambda function requirements
+
+You Lambda function will be invoked by the LCA ChimeVC CallTranscriber function when a new call is received. The full call START event from ChimeVC, which includes the SIPREC invite headers, is passed as the input event to your Lambda. 
```

Comment:
This should probably mention that the function will only execute after both Agent and Caller EventBridge events are received, and isCaller should be true by default?